src/read.c

```diff
@@ -3588,27 +3588,30 @@ static avifDecoderItem * avifDecoderDataFindColorItem(avifDecoderData * data)
     return NULL;
 }
 
+// Returns AVIF_TRUE if item is an alpha auxiliary item of the parent color
+// item.
 static avifBool avifDecoderItemIsAlphaAux(avifDecoderItem * item, uint32_t colorItemId)
 {
     if (item->auxForID != colorItemId)
         return AVIF_FALSE;
-    // Is this an alpha auxiliary item of the color item?
     const avifProperty * auxCProp = avifPropertyArrayFind(&item->properties, "auxC");
     return auxCProp && isAlphaURN(auxCProp->u.auxC.auxType);
 }
 
 // Finds the alpha item whose parent item is colorItem and sets it in the alphaItem output parameter. Returns AVIF_RESULT_OK on
 // success. Note that *alphaItem can be NULL even if the return value is AVIF_RESULT_OK. If the colorItem is a grid and the alpha
-// item is represented as a set of auxl items to each color tile, then a fake item will be created and isAlphaItemInInput will be
-// set to AVIF_FALSE. Otherwise, isAlphaItemInInput will be set to AVIF_TRUE when *alphaItem is not NULL.
+// item is represented as a set of auxl items to each color tile, then a fake item will be created and *isAlphaItemInInput will be
+// set to AVIF_FALSE. In this case, the alpha item merely exists as a pointer to hold the locations of the alpha grid items. The
```

Comment:
1. Nit: The word "pointer" is a little unclear here. Maybe we can remove "as a pointer"?

2. alpha grid items => alpha tile items ?

Reply:
Done.